sklearn/decomposition/pca.py

```diff
@@ -478,6 +512,9 @@ def score(self, X, y=None):
         return np.mean(self.score_samples(X))
 
 
+@deprecated("it will be removed in 0.19. Use PCA(svd_solver='randomized') "
```

Comment:
I was trying to avoid a repetition. Currently the deprecation message is 
```
DeprecationWarning: Class RandomizedPCA is deprecated; it will be removed in 0.19. Use PCA(svd_solver='randomized') instead. The new implementation DOES NOT store whiten components_. Apply transform to get them.
  warnings.warn(msg, category=DeprecationWarning)
```

Although, I have checked in other places in the code base and to be consistent I should do what you are suggesting. Thanks.